sdks/python/apache_beam/runners/portability/local_job_service.py

```diff
@@ -169,7 +169,7 @@ def GetMessageStream(self, request, context=None):
         resp = beam_job_api_pb2.JobMessagesResponse(message_response=msg)
       yield resp
 
-  def DescribePipelineOptions(self, request, context=None):
+  def DescribePipelineOptions(self, request, context=None, timeout=None):
```

Comment:
It doesn't look like the `timeout` argument is actually used -- see other comment below